### ISSUE TITLE
New Feature: Slow-motion mode

### ISSUE THREAD

**Issue by nicolasmafra** [panel-attack/panel-game#44]
A slow-motion game mode helps in training to make long chains and in recognizing new chain patterns. It would be simple to develop too.

**Comment by JamesVanBoxtel** [comment]
How is this different then just playing on a low level? Are you saying slower than level 1?

**Comment by nicolasmafra** [comment]
Yes, slower than level 1. This could help beginners learn to make long chains.
